components/ai_chat/resources/page/components/delete_conversation_modal/index.tsx

```diff
@@ -0,0 +1,76 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import Button from '@brave/leo/react/button'
+import Dialog from '@brave/leo/react/dialog'
+
+// Utils
+import { getLocale } from '$web-common/locale'
+
+// Hooks
+import { useAIChat } from '../../state/ai_chat_context'
+
+// Styles
+import styles from './style.module.scss'
```

Comment:
nit: prefer to explicitly order outside-in instead of by type. nit because this is the correct order coincidentally.

Reply:
Updated with https://github.com/brave/brave-core/compare/8b040a9ebbb6c5083241f7b9f37c228ce047b03f..e0c4e315571ec1a6a11dd7511dafca99afade81c